code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -512,3 +512,53 @@
 	if(index == 1 && target.address == text && target.active)
 		if(target.wearer && target.wearer.stat != DEAD)
 			return target
+
+/*
+	Succour
+	Heals another person, quite powerfully. Only works on NT disciples
+*/
+/datum/ritual/targeted/cruciform/priest/heal_other
+	name = "Succour"
+	phrase = "Venite ad me, omnes qui laboratis, et onerati estis et ego reficiam vos"
+	desc = "Heal a nearby disciple"
+	cooldown = TRUE
+	cooldown_time = 100
```

Comment:
are you sure this is 100 seconds and not 10 seconds, because i'm pretty sure byond measures time in deciseconds by default

Reply:
No idea, this was the default cooldown I copy pasted from inquisitor.dm.